build/images/scripts/install_cni

```diff
@@ -46,10 +46,20 @@ if [[ ! " ${binaries[*]} " =~ " bandwidth " ]]; then
   install -m 755 /opt/cni/bin/bandwidth /host/opt/cni/bin/bandwidth
 fi
 
+FILE_MODE="0644"
+if [[ -n "${CONFIG_FILE_MODE}" ]]; then
+  if [[ "${CONFIG_FILE_MODE}" =~ ^0?[0-7]{3}$ ]]; then
+    FILE_MODE=${CONFIG_FILE_MODE}
+  else
+    echo "Warning: Invalid CONFIG_FILE_MODE."
+    FILE_MODE=${CONFIG_FILE_MODE}
```

Comment:
if CONFIG_FILE_MODE is invalid, we shld not be assigning it !

Reply:
As @tnqn suggested in another thread, we will let the deployment fail and users will be responsible to fix the invalid mode.